storage/wdpost_run.go

```diff
@@ -113,11 +155,27 @@ func (s *WindowPoStScheduler) checkNextRecoveries(ctx context.Context, dlIdx uin
 	ctx, span := trace.StartSpan(ctx, "storage.checkNextRecoveries")
 	defer span.End()
 
+	var sm *types.SignedMessage
```

Comment:
Should probably change `sm, err := s.api.MpoolPushMessage` below to set with `=`

Reply:
Oops, this was a merge error, thanks for catching!

Reply:
Actually, the `sm, err := s.api.MpoolPushMessage` below _does_ set on this variable. It is what the spec calls a "redeclaration": https://golang.org/ref/spec#Short_variable_declarations. But I will move this logic to `runPost`, and make `checkNextRecoveries` and `checkNextFaults` return the data we need. That way we can also log if an error happens.